### DISCUSSION TITLE
What is the release strategy for this library?

### DISCUSSION THREAD

**Discussion by ryan-dahlia** [juliencrn/usehooks-ts#425] [Help] [answered]
Hey there! Amazing library, thank you so much to @juliencrn and all the contributors!

We are a little confused about the release strategy. Dependabot has been pinging us about new versions since v2.9.2 - v.2.94, but we see v2.9.1 as the most recent release in Github. 

Are these latest releases stable ones or something in testing? Thanks!

**Comment by juliencrn** [comment, marked as answer]
Hi @ryan-dahlia, 
It was un-synchronized for a moment but now fixed. 
Right now `2.9.5` is the latest (on both npm and github 😂)

**Reply by ryan-dahlia** [reply]
Ok sounds good, thank you for clarifying!

So is 2.9.5 a stable release?
